spec/zoneddatetime.html

```diff
@@ -1078,18 +1078,40 @@ <h1>Properties of Temporal.ZonedDateTime Instances</h1>
   <emu-clause id="sec-temporal-zoneddatetime-abstract-ops">
     <h1>Abstract operations</h1>
 
-    <emu-clause id="sec-temporal-interpretisodatetimeoffset" aoid="InterpretISODateTimeOffset">
-      <h1>InterpretISODateTimeOffset ( _year_, _month_, _day_, _hour_, _minute_, _second_, _millisecond_, _microsecond_, _nanosecond_, _offsetBehaviour_, _offsetNanoseconds_, _timeZone_, _disambiguation_, _offsetOption_, _matchBehaviour_ )</h1>
-      <p>
-        The abstract operation InterpretISODateTimeOffset determines the exact time in _timeZone_ corresponding to the given calendar date and time, and the given UTC offset in nanoseconds.
-        In the case of more than one possible exact time, or no possible exact time, an answer is determined using _offsetBehaviour_ (one of ~option~, ~exact~, or ~wall~), _disambiguation_ (one of *"earlier"*, *"later"*, *"compatible"*, or *"reject"*) and _offsetOption_ (one of *"ignore"*, *"use"*, *"prefer"*, or *"reject"*).
-      </p>
-      <p>
-        As a special case when parsing ISO 8601 strings which are only required to specify time zone offsets to minutes precision, if _matchBehaviour_ is ~match minutes~, then a value for _offsetNanoseconds_ that is rounded to the nearest minute will be accepted in those cases where _offsetNanoseconds_ is compared against _timeZone_'s offset.
-        If _matchBehaviour_ is ~match exactly~, then this does not happen.
-      </p>
+    <emu-clause id="sec-temporal-interpretisodatetimeoffset" type="abstract operation">
+      <h1>
+        InterpretISODateTimeOffset (
+          _year_: an integer,
+          _month_: an integer,
+          _day_: an integer,
+          _hour_: an integer,
+          _minute_: an integer,
+          _second_: an integer,
+          _millisecond_: an integer,
+          _microsecond_: an integer,
+          _nanosecond_: an integer,
+          _offsetBehaviour_: one of ~option~, ~exact~ or ~wall~,
+          _offsetNanoseconds_: an integer,
+          _timeZone_: an Object,
+          _disambiguation_: one of *"earlier"*, *"later"*, *"compatible"*, or *"reject"*,
+          _offsetOption_: one of *"ignore"*, *"use"*, *"prefer"*, or *"reject"*,
```

Comment:
should these be converted from strings to spec enums earlier in the process? (unrelated to this PR)